provd/proto/keyboard.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package keyboard;
+
+service KeyboardService {
```

Comment:
My understanding is that the flutter frontend will already have access to all the keyboard types, so I have left the get out of this definition set.

Reply:
It'd be nice to eventually remove those from the frontend though and be able to get a list of available keyboards via `GetKeyboard()` from the backend. Ideally we'd also have a keyboard detection mechanism like subiquity does, so I'd include `GetKeyboardStep()` in the API (of course we don't have to implement it immediately :D)

Reply:
I guess it falls in next to the networking logic. Ideally should be on the be, but pragmatically might need to stay on the fe for the time being :sweat_smile: